### PR TITLE
Fix Issue 8207 | [Bug] Shell Flyout Items on UWP aren't showing the Title

### LINKED ISSUE #8207: [Bug] Shell Flyout Items on UWP aren't showing the Title
## Description

https://github.com/xamarin/Xamarin.Forms/pull/6015#issuecomment-546010203

Hi, thanks for the good work on this, been waiting for Shell on UWP for a while.

I just switched my xamarin forms app to enable shell on the UWP.
My menu is working on iOS and Android but on UWP renders as below
image

I'm using this syntax:-
```
<FlyoutItem Title="Dashboard">
        <FlyoutItem.Icon>
            <FontImageSource Color="Black" Glyph="{x:Static fa:FAIcons.chart}" FontFamily="{StaticResource FontAwesomeSolid}" Size="30"/>
        </FlyoutItem.Icon>
        <Tab>
            <ShellContent Title="Charts" ContentTemplate="{DataTemplate views:ChartsPage}"/>
        </Tab>
</FlyoutItem>
```
Am I using something thats not supported?
All NuGet packages are updated to latest stable.

Thanks

### Steps to Reproduce

- Create Xamarin Forms Shell Template
- Add UWP platform
- you'll see the issue described above

### Expected Behavior
The flyout should show the text of each item not the object type

### Actual Behavior

Flyout just shows the object Type

### Basic Information

- Version with issue:
- Platform Target Frameworks: 
  - UWP:  16299

### PR DESCRIPTION
**Pull request by felipebaltazar**
### Description of Change ###
I've create an override to ` SelectTemplateCore(object item, DependencyObject container)`

### Issues Resolved ### 

- fixes #8207

### API Changes ###
Added to Xamarin.Forms.Platform.UWP.ShellFlyoutTemplateSelector:
 - Windows.UI.Xaml.DataTemplate SelectTemplateCore(object item, DependencyObject container)

### Platforms Affected ### 
- UWP

### Behavioral/Visual Changes ###
FlyoutItems will show title and icons when it setted

### Before/After Screenshots ### 
- Before

<p align="center">
	<kbd>
		image
	</kbd>
</p>

- After

<p align="center">
	<kbd>
		image
	</kbd>
</p>

### Testing Procedure ###
1. Create Xamarin Forms Shell Template
2. Add some flyout items
3. Add and run UWP platform 

> Or just run the Issue8207 test from Control Gallery

### PR Checklist ###

- [ ] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

**Comment by felipebaltazar** [comment]
Hello!

How can I create, from C# code, a layout like this:
<p align="center">
	<kbd>
		image
	</kbd>
</p>

I'm trying to write some tests for these scenarios, but I founded only Xaml examples

**Comment by jfversluis** [comment]
Is there any specific reason you want to do it in code @felipebaltazar ? You should be able to use XAML in your issue page. See other examples like Issue4684.

**Comment by felipebaltazar** [comment]
> Is there any specific reason you want to do it in code @felipebaltazar ? You should be able to use XAML in your issue page. See other examples like Issue4684.
> 
> Of course you should also be able to do it in code. You can find some examples of that in other issue pages as well.

@jfversluis  This is true! I went through some problems the first time I tried to create a Xaml-based UITest ..
But in other PRs that I created (in this repository), I managed to solve them ... I will recreate this problem test in Xaml.

**Comment by felipebaltazar** [comment]
I've updated the issue class and added a UI test

**Comment by felipebaltazar** [comment]
@rmarinho  @samhouts  Any position about this PR? Failing tests seems like not related

**Comment by felipebaltazar** [comment]
I've updated this branch, can we merge this?

**Comment by felipebaltazar** [comment]
@rmarinho @samhouts  Any position about this PR ?

File: Xamarin.Forms.Platform.UAP/Shell/ShellRenderer.cs [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.ComponentModel;
+using System.Linq;
```
Comment (samhouts) [inline Xamarin.Forms.Platform.UAP/Shell/ShellRenderer.cs]:
```suggestion
```

**Pull request merged**